extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java

```diff
@@ -493,7 +493,7 @@ public void run() {
 
         @Override
         public boolean test(DotName t) {
-            return Classes.isPrimitive(t.toString()) || t.toString().startsWith("java.net.");
+            return Classes.isPrimitive(t.toString()) || t.toString().startsWith("java.");
```

Comment:
And maybe we should handle the primitives in `DefaultIgnorePredicate` directly.

/cc @geoand

Reply:
Makes sense

Reply:
Ah yes, good catch. That's a good idea

Reply:
I'll provide a PR

Reply:
Sounds like general solution.